src/pages/signup/signup.styles.js

```diff
@@ -0,0 +1,7 @@
+import styled from 'styled-components';
+
+export const Link = styled.div`
```

Comment:
Since this is a Link component, this should not be a `div` element, we should try to use the correct HTML elements. We should use react router dom Link component here:
```suggestion
import { Link as ReactRouterDomLink } from 'react-router-dom';

export const Link = styled(ReactRouterDomLink)`
```